kafka/cc/avn_KafkaConsumer_Simple.cc

```diff
@@ -0,0 +1,60 @@
+#include <kafka/KafkaConsumer.h>
+
+#include <cstdlib>
+#include <iostream>
+#include <signal.h>
+#include <string>
+
+#include "config.hh"
+
+std::atomic_bool running = {true};
+
+void stopRunning(int sig) {
+    if (sig != SIGINT) return;
+
+    if (running) {
+        running = false;
+    } else {
+        // Restore the signal handler, -- to avoid stuck with this handler
+        signal(SIGINT, SIG_IGN); // NOLINT
+    }
+}
+
+int main()
+{
+    using namespace kafka;
+    using namespace kafka::clients::consumer;
+
+    // Use Ctrl-C to terminate the program
+    signal(SIGINT, stopRunning);    // NOLINT
+
+    // Create a consumer instance
+    KafkaConsumer consumer(props);
+
+    // Subscribe to topics
+    consumer.subscribe({topic});
+
+    while (running) {
+        // Poll messages from Kafka brokers
+        auto records = consumer.poll(std::chrono::milliseconds(100));
+
+        for (const auto& record: records) {
+            if (!record.error()) {
+                std::cout << "Got a new message..." << std::endl;
+                std::cout << "    Topic    : " << record.topic() << std::endl;
+                std::cout << "    Partition: " << record.partition() << std::endl;
+                std::cout << "    Offset   : " << record.offset() << std::endl;
+                std::cout << "    Timestamp: " << record.timestamp().toString() << std::endl;
+                std::cout << "    Headers  : " << toString(record.headers()) << std::endl;
+                std::cout << "    Key   [" << record.key().toString() << "]" << std::endl;
+                std::cout << "    Value [" << record.value().toString() << "]" << std::endl;
+            } else {
+                std::cerr << record.toString() << std::endl;
```

Comment:
I guess we need to dump some sort of error message here? Maybe even bail out early?